pkgs/development/tools/wally-cli/default.nix

```
@@ -1,28 +1,28 @@
-{ lib, buildGoPackage, fetchFromGitHub, pkg-config, libusb1 }:
+{ lib, buildGoModule, fetchFromGitHub, pkg-config, libusb1 }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "wally-cli";
-  version = "1.1.1";
+  version = "2.0.0-linux";
 
-  goPackagePath = "github.com/zsa/wally";
-  subPackages = [ "cli" ];
+  goPackagePath = "github.com/zsa/wally-cli";
+  subPackages = [ "." ];
 
   nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [ libusb1 ];
 
   src = fetchFromGitHub {
     owner = "zsa";
-    repo = "wally";
-    rev = "68960e452ee0f6c7142f5008d4b1cdc6284d3de7";
-    sha256 = "122m5v7s5wqlshyk2salmd848lqs4rrz54d2ap11ay61kijm0bs2";
+    repo = "wally-cli";
+    rev = version;
```

So to me it makes sense to set the tag properly without having the platform in the version.

```suggestion
    rev = "${version}-linux";
```